index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Raj Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <main class="rj">
+    <header>
+      <div>      <a href="#" class="logo">My Logo</a></div>
+
+      <div>
+        <img src="image/Union.png" alt="button" />
+      
+    </div>
+
+    </header>
+    <section>
+      <h1>I'm Vivek Raj glad to see you !</h1>
+      <p class="software">
```

Comment:
- [ ] There seems to be a slightly larger spacing between the header section and the heading section which is what @roshan-bajgain  mentioned [here](https://github.com/VivekRaaZ/portfolio/pull/1#discussion_r1257062807). Kindly reduce the paddings/ margins used to separate these two sections by a small amount to ensure that your design look more aesthetic by having a uniform design throughput.

**This is how the section looks like on my end**

![image](https://github.com/VivekRaaZ/portfolio/assets/86318284/d1f622be-f819-4cf4-9ded-38fa51765b97)

**This is the desired design as provided in the Figma design:**

![image](https://github.com/VivekRaaZ/portfolio/assets/86318284/b219aa89-65c3-4a49-b37b-a7c94a7a14cd)

**Notice the slight design differences?**